### PR TITLE
[feat] added Slider

### PR DESCRIPTION
**Pull request by omjadhav85**
added Slider component

image

### CODE REVIEW

File: routes/slider.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+              <input
+                type="range"
+                min="0"
+                max="100"
+                step="10"
+                value="33"
+                list="tickmarks"
+                class="slider"
+              />
+              <datalist id="tickmarks" class="slider-ticks">
```
Comment (Hamdrive) [inline routes/slider.html:85]: You can additionally add values which will help user to see where breakpoints are on your slider

**Review by Hamdrive — COMMENTED** [review]
Great implementation of slider component, you have also implemented hashmarks which will help user know various breakpoints
Have made suggestion for labels as well for slider

File: routes/slider.html (hunk 1 above)
Comment (omjadhav85) [reply routes/slider.html:85]: Thanks for the suggestion Hamza...added labels as well!

**Review by krate27 — APPROVED** [review]
Great Work Done.
-Clean folder structure.
-You made sure everything is used in slider, hashmarks, options, label.
- Good to go from my side.

File: routes/slider.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            </li>
+            <li class="list-item">
+              <a href="./modal.html">Modal</a>
+            </li>
+            <li class="list-item active">
+              <a href="./slider.html">Slider</a>
+            </li>
+          </ul>
+        </aside>
+        <section class="content">
```
Comment (Jia-A) [inline routes/slider.html:66]: Nice use of semantic HTML throughout the code.

**Review by Jia-A — APPROVED** [review]
- Liked the relevant class names.
- Use of semantic HTML is good.
- Structured code is there which is a nice practice. 
- Good to merge.

**Comment by omjadhav85** [comment]
Thanks everyone! Merging this PR

**Pull request merged**